src/sec_edgar_bulker/core/proxy_manager.py

```
@@ -0,0 +1,256 @@
+    def load_proxies(self) -> None:
+        """Load proxies from file.
+        
+        Raises:
+            NoProxiesAvailableError: If required proxies are not available
+            InvalidProxyFormatError: If proxy format is invalid
+            ProxyFileError: If proxy file cannot be read
+        """
+        try:
+            proxy_file = Path(self.settings.file)
+            if not proxy_file.exists():
+                self.logger.error(f"Proxy file not found: {proxy_file}")
```

Ruff (0.8.2)

Logging statement uses f-string

(G004)